service/history/queues/dlq_writer_test.go

```diff
@@ -126,6 +126,8 @@ func TestDLQWriter_ErrGetNamespaceName(t *testing.T) {
 	counter, ok := recordings[0].Value.(int64)
 	assert.True(t, ok)
 	assert.Equal(t, int64(1), counter)
+	assert.Len(t, recordings[0].Tags, 1)
+	assert.Equal(t, recordings[0].Tags[metrics.TaskCategoryTagName], task.GetCategory().Name())
```

Comment:
optional/debatable nit: I generally prefer to use literals whenever possible in tests assertions since I've made the mistake more than once to provide the wrong data and didn't notice since the test just passed.

Reply:
That is good idea, Thanks!